Readme.md

```diff
@@ -1,29 +1,27 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128592237/22.2.4%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E622)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# XAF - How to prevent a Detail View from being shown for a List View record
+# XAF - Prevent Showing a Detail View of a List View Object
 
-## Scenario
+This example demonstrates how to prevent a user from invoking a Detail View of an object from a List View.
 
-For certain data forms, a developer needs to limit end-users to editing only through the List View, i.e. without invoking a separate Detail View. This is usually done by activating the inline editing and `MasterDetailMode` = `ListViewAndDetailView` features as described in the following article: [List View Edit Modes](https://docs.devexpress.com/eXpressAppFramework/113249/ui-construction/views/list-view-edit-modes).
+To allow users to edit objects directly in a List View, enable the [in-place editing](https://docs.devexpress.com/eXpressAppFramework/113249/ui-construction/views/list-view-edit-modes#in-place-editing) and [split layout](https://docs.devexpress.com/eXpressAppFramework/113249/ui-construction/views/list-view-edit-modes#split-layout-the-masterdetailmode-property) functionality.
```

Comment:
I will not count this as your issue, however, this way it would have been better:

> the in-place edit and split layout functionality.

"in-place editing" as a standalone proper noun sounds fine, but when you use it as a compound adjective for "functionality", it sounds better in a non-gerund form -- "in-place edit functionality".

I hope this explanation makes sense.

```suggestion
To allow users to edit objects directly in a List View, enable the [in-place edit](https://docs.devexpress.com/eXpressAppFramework/113249/ui-construction/views/list-view-edit-modes#in-place-editing) and [split layout](https://docs.devexpress.com/eXpressAppFramework/113249/ui-construction/views/list-view-edit-modes#split-layout-the-masterdetailmode-property) functionality.
```